docs/patterns/value-objects.md

```
@@ -1,120 +1,140 @@
-
-### ✅ Uso em DTOs e Actions:
-
-    Utilize VOs em DTOs para garantir que dados inválidos nunca cheguem à camada de integração ou negócio.
+```php
+// src/ValueObjects/Structured/Discount.php
 
-### ✅ Formato separado do valor:
+namespace AsaasPhpSdk\ValueObjects\Structured;
 
-    Se precisar exibir de forma amigável, implemente FormattableContract e não altere o valor interno.
+use AsaasPhpSdk\ValueObjects\Base\AbstractStructuredValueObject; // 1. Herda da base
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)